packages/react-components/react-breadcrumb/stories/Breadcrumb/BreadcrumbWithTooltip.stories.tsx

```diff
@@ -133,10 +133,10 @@ const MenuWithTooltip = (props: PartitionBreadcrumbItems<Item>) => {
   if (!isOverflowing && overflowItems && overflowItems.length === 0) {
     return null;
   }
-  const overflowItemsLength = overflowItems?.length ?? 0;
+  const overflowItemsCount = overflowItems ? overflowItems.length + overflowCount : overflowCount;
```

Comment:
**suggestion:** looks like good candidate to encapsulate this logic to generic function as it's duplicated in previous story